sse/sse_handler.go

```diff
@@ -5,22 +5,23 @@ import (
 	"io"
 	"time"
 
+	log "github.com/sirupsen/logrus"
+
 	"github.com/allegro/marathon-consul/events"
 	"github.com/allegro/marathon-consul/marathon"
 	"github.com/allegro/marathon-consul/metrics"
-	log "github.com/sirupsen/logrus"
 )
 
-// SSEHandler defines handler for marathon event stream, opening and closing
+// HandlerSSE defines handler for marathon event stream, opening and closing
```

Comment:
why this rename?

Reply:
Linter error: `sse/sse_handler.go:17:6: type name will be used as sse.SSEHandler by other packages, and that stutters; consider calling this Handler (golint)`
Unfortunately Handler is already taken by a func in `sse/sse.go`